scripts/setup_container_runtime.sh

```diff
@@ -2,6 +2,7 @@
 # vim: ts=4 sw=4 expandtab
 function setup_container_runtime () {
   loginctl enable-linger "$(id -nu)"
+  (podman system info > /dev/null) || podman system reset --force
```

Comment:
Why the parens around `podman system info > /dev/null`?

Reply:
no special reason. removed.